installers/metrics/ansible/roles/pgo-metrics-preflight/tasks/preflight-alertmanager.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Check if inventory file variables are defined for Alertmanager
+  tags: always
+  fail:
+    msg: "Please specify the value of {{item}} in your inventory file"
+  when: lookup('vars', item, default='') == ''
+  loop:
```

Comment:
Thinking we should also check for a valid image prefix, name and tag here.

Reply:
ah i missed that. Ill add in checks for those things. Looks like `preflight-grafana.yml` only checks for name and prefix. Is there a reason why it wouldn't need a check for tags?